references.json

```diff
@@ -58,6 +58,12 @@
     "authors": ["Mike West"],
     "publisher": "W3C"
   },
+  "CLIENT-HINTS" : {
+    "href" : "http://httpwg.org/http-extensions/client-hints.html",
```

Comment:
Is https://tools.ietf.org/html/draft-ietf-httpbis-client-hints perhaps a better reference since it uses HTTPS and that's typically what we use to reference RFCs? I guess I don't care strongly, just wondering.